CMakeLists.txt

```diff
@@ -117,18 +122,32 @@ else()
   set_property(CACHE LOGIT_BUILD_OPTION PROPERTY STRINGS LOGIT_AS_INT_SRC LOGIT_AS_EXT_SHARED LOGIT_AS_EXT_STATIC)
   message(STATUS "LogIt build option LOGIT_BUILD_OPTION [${LOGIT_BUILD_OPTION}]")
 
+  #
+  # Clone LogIt if needed
+  # 
+  if("${LOGIT_BUILD_OPTION}" STREQUAL "LOGIT_AS_INT_SRC")
+  	clone_LogIt()
```

Comment:
Aaargh, I hate the way github restricts which lines I can add comments to in these reviews (comment should be to line 55 I think - declaration of clone_LogIt function)

Why not change ```clone_Logit``` to ```fetch_LogIt``` using FetchContent_*() in this PR too. It keeps the build file clean and homogeneous. The only reason LogIt was cloned the way it is (i.e. the same way open62541 was cloned prior to this PR) was because at the time cmake3 was apparently not allowed (the epel repo was deemed not OK).

If FetchContent is on the menu now then let's feast!